src/browser/rvm/rvm_message_bus.ts

```diff
@@ -278,7 +278,7 @@ export class RVMMessageBus extends EventEmitter  {
         const payload: any = Object.assign({
             processId: process.pid,
             runtimeVersion: processVersions.openfin,
-            securityRealm: argo ? argo['security-realm'] : ''
+            securityRealm: argo ? (argo['security-realm'] || '') : ''
```

Comment:
No need to check if `argo` is defined; it's one of the first things defined in core_state. `argo['security-realm'] || ''` should suffice.

Reply:
It's not defined in tests, so I added the extra check it in the original PR, #653